### ISSUE TITLE
Write Wingsail Publishing Logic

### ISSUE THREAD

**Issue by DFriend01** [UBCSailbot/controller#12]
### Purpose

In order for other parts of the software system to use the output(s) from the wingsail controller, the results need to be "published" to the ROS network. The idea is to periodically send new data out to the ROS network at some frequency.

There is no publishing logic currently implemented in the wingsail node. The purpose of this issue is to publish the trim tab angle to the ROS network.

See the **Resources** section to learn more about the background tools that will be needed to successfully resolve this issue.

#### Defining the Publishing Topic Type

Each ROS topic has an associated topic type, and right now, there is no defined topic type for the trim tab angle. In addition to writing the publishing logic, the topic type definition will need to be written in our [custom interfaces repository](https://github.com/UBCSailbot/custom_interfaces). In particular, the following will need to be done:

1. Define the trim tab angle type in the `msg` directory (see other topic type definitions for examples).
2. Add the topic type definition to `CMakeLists.txt`
3. Open a pull request in the custom interfaces repository

While the pull request is under review, using the branch used to define the trim angle topic type for building is fine until it is merged.

#### Writing the Publishing Logic

Modify the function `__init_publishers` in the `WingsailControllerNode` class located in [wingsail_ctrl_node.py](https://github.com/UBCSailbot/controller/blob/main/controller/wingsail/wingsail_ctrl_node.py). One publisher should be defined, which is the publisher for the trim tab angle.

In addition to defining the publishers, it is also necessary to implement callbacks that periodically publish the trim tab angle. Define a new function called `__init_timer_callbacks` and define the callbacks there (remember to call this function in the `__init__` function). It's okay to publish dummy data for now (i.e. zero) -- we just want to get the publishing working. We can worry about implementing the "smart" logic that actually computes the trim tab angle at a later time.

For an example that was already done, check out the [physics engine node](https://github.com/UBCSailbot/boat_simulator/blob/main/boat_simulator/nodes/physics_engine/physics_engine_node.py), and in particular, look at its `__init_publishers` and `__init_timer_callbacks` functions.

#### Testing

In each timer callback, write an info log to stdout confirming that new data has been published. You should observe this log when running the controller node. Copy and paste the output in the pull request showing the ROS info logs.

Once again, see the physics engine node as an example of how to do this.

### Resources

- [ROS2 Basic Concepts](https://docs.ros.org/en/humble/Concepts/Basic.html): Only up to (and including) ROS Topics is mandatory to resolve this issue. However, you are encouraged to reach through the entire **Basic Concepts** section to learn about how ROS2 works.
- [GitHub Development Worflow](https://ubcsailbot.github.io/docs/latest/reference/github/workflow/overview/): Read through this before starting on the issue if you have not already.
- [Sailbot Python Conventions](https://ubcsailbot.github.io/docs/latest/reference/python/conventions/): Read through this before starting on the issue if you have not already.

**Comment by DFriend01** [comment]
Resolved with https://github.com/UBCSailbot/controller/pull/15
